Automattic/simplenote-macos
# Interlinks Autocomplete UI: Key Exclusions

### Details:
In this PR we're adjusting the keystrokes accepted by the Interlink Autocomplete Window, so that whenever **Shift (or) CMD (or) Control** are present in any of the accepted keystrokes, we actually dismiss the window.

@danielebogo sir!! can I trouble you with this one?
Thanks in advance!!

Ref. #663

### Test
1. Open a new document
2. Type `[Keyword` (with `keyword` being a word that's present in the title of any other note

- [ ] Verify the Interlink Autocomplete Window shows up
- [ ] Verify that pressing **CMD + SHIFT + Arrow UP** lets you select the whole document (and dismisses the Autocomplete Window)

### Release
These changes do not require release notes.
